Solutions/Alibaba Cloud/DataConnectors/AliCloudSentinelConnector/__init__.py

```diff
@@ -40,7 +40,7 @@
 def generate_date():
     current_time = datetime.utcnow().replace(second=0, microsecond=0) - timedelta(minutes=10)
     state = StateManager(connection_string=connection_string)
-    past_time = state.get()
+    past_time = datetime.strptime("30 Nov 00", "%d.%m.%Y %H:%M:%S")
```

Comment:
@v-sabiraj : Why is the date hard coded?

Reply:
@v-rucdu, updated that, thanks.